### PR TITLE
Multiple registration urls

### PR DESCRIPTION
**Pull request by Chilipp**
@eileen-kuehn, this PR allows multiple registration urls. If you enter the registration urls as

```yaml
registration_url:
    - https://indico.scc.kit.edu/event/923/
    - https://indico.scc.kit.edu/event/922/
```

the button becomes a dropdown menu

image

We can do better than that for sure, but it does the job at the moment

- [x] revert e261e83dfd23f72bf303cefca7c507d1eb8ac71c

### CODE REVIEW

File: _includes/registration-button.html [hunk 1]
```diff
@@ -1,7 +1,16 @@
 {% if page.registration_url %}
+{% if page.registration_url[0] %}
+<select class="btn btn--success registration-menu" >
+    <option value="" selected disabled hidden>&#xf56b; Register</option>
+    {% for uri in page.registration_url %}
+        <option value="{{ uri }}">Occurence #{{ forloop.index }}</option>
```
Comment (eileen-kuehn) [inline _includes/registration-button.html:6]:
```suggestion
        <option value="{{ uri }}">Occurrence #{{ forloop.index }}</option>
```

**Review by eileen-kuehn — CHANGES_REQUESTED** [review]
Great work, thanks!
Unfortunately, the button does not work for me (client: Safari, Chrome). When clicking on it, the dropdown appears so I can never trigger the functionality of calling the URL for registration.

File: _includes/registration-button.html (hunk 1 above)
Comment (Chilipp) [reply _includes/registration-button.html:6]: sure, there is no functionality yet implemented, sorry, forgot to mention this :sweat_smile:

Comment (Chilipp) [reply _includes/registration-button.html:6]: I can do it this evening, but I already did want to show it to you to get your feedback about the appearance and the basic implementation

File: _events/workshops/event-017.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ time:
 prerequisites: "no specific knowledge/skills required"
 date: 2020-08-18
 last_modified_at: 2020-09-08
+registration_url:
+    - https://indico.scc.kit.edu/event/923/
+    - https://indico.scc.kit.edu/event/922/
```
Comment (eileen-kuehn) [inline _events/workshops/event-017.md:37]:
```suggestion
    - https://indico.scc.kit.edu/event/924/
    - https://indico.scc.kit.edu/event/925/
```

Comment (eileen-kuehn) [reply _events/workshops/event-017.md:37]: I just created the events, so we can pick the correct registration urls already.

**Comment by Chilipp** [comment]
alright, this is ready to be reviewed @eileen-kuehn 

see here for a preview: https://1212-267395254-gh.circle-artifacts.com/0/SORSE/programme/workshops/event-017/index.html

**Comment by Chilipp** [comment]
@eileen-kuehn. did you had the chance to look at https://1212-267395254-gh.circle-artifacts.com/0/SORSE/programme/workshops/event-017/index.html whether you are satisfied with the method? If yes, we can merge this and #318

**Review by eileen-kuehn — APPROVED** [review]
Looks great, thanks.
Only thing we might consider is adapting the width of the button to the textwidth, but this is only very minor ;)
So, good to go 👍

**Pull request merged**